modules/sdk/project-templates/project-templates-rest/src/main/resources/archetype-resources/build.gradle

```diff
@@ -17,9 +17,11 @@ dependencies {
 #if (${dependencyManagementEnabled} == "true")
 	compileOnly group: "javax.ws.rs", name: "javax.ws.rs-api"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations"
+    compileOnly group: "org.osgi", name: "org.osgi.service.jaxrs", version: "1.0.0"
```

Comment:
Hey @sez11a for this one we don't want to put a version.  This version will be provided by a new BOM once this project template gets merged.